LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Firebase, Inc.
```

Comment:
This should be Google LLC.

Reply:
My bad, I double checked and to me it seemed that Firebase, Inc was still in existence as a subsidiary of Google LLC.

Reply:
No problem, this is Google policy but it's not published publicly so there was no way for you to know what the correct name should be.

Reply:
> No problem, this is Google policy but it's not published publicly so there was no way for you to know what the correct name should be.

Ah that makes sense. Either way, glad to be of help. It's a pedantic change, but I noticed it when I wanted to add proper attributions in my app by copying the license. Figured no one noticed after init three years ago.